tests/bulkwritecommand/bulkwritecommand-debug-001.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\BulkWriteCommand debug output
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+$bulk = new MongoDB\Driver\BulkWriteCommand([
+    'bypassDocumentValidation' => true,
+    'comment' => 'foo',
+    'let' => ['foo' => 1],
+    'ordered' => true,
+    'verboseResults' => true,
+]);
+
+var_dump($bulk);
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+object(MongoDB\Driver\BulkWriteCommand)#%d (%d) {
+  ["bypassDocumentValidation"]=>
+  bool(true)
+  ["comment"]=>
+  string(3) "foo"
+  ["let"]=>
+  object(stdClass)#%d (%d) {
+    ["foo"]=>
+    int(1)
+  }
+  ["ordered"]=>
+  bool(true)
+  ["verboseResults"]=>
+  bool(true)
+  ["session"]=>
+  NULL
+}
```

Comment:
The `BulkWriteCommand` doesn't expose the list of operations when using `var_dump` or xdebug. There is no way to inspect its contents without executing it and using a subscriber to get the raw command.

I think the `executed` property should also be exposed.

Reply:
Adding `executed` would make sense if we have the information already. As the operations list could potentially be big, I'd rather not expose it in the debug output.

Reply:
Tracked in [PHPC-2582](https://jira.mongodb.org/browse/PHPC-2582)